scripts/x-compile-android-amd64.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e
+
+PLATFORM=android
+SCRIPTS=android
+LIBSUFFIX=android_amd64
+
+echo "Building for: $PLATFORM"
+
+echo "Bootstrapping Android NDK"
+
+$ANDROID_NDK_ROOT/build/tools/make_standalone_toolchain.py --install-dir=/usr/$ANDROID_CHAIN_AMD64 --api=21 --arch=x86_64
+
+
+echo BUILD DEPS
+pushd $DEP_DIR
+rm -rf asio* lz4* mbedtls* #lzo* boost* minicrypto openssl* polarssl* snappy*
+echo "******* ASIO"
+$O3/core/deps/asio/build-asio
+echo "******* MBEDTLS"
+TARGETS=android-amd64 $O3/core/scripts/android/build-mbedtls
+echo "******* LZ4"
+TARGETS=android-amd64 $O3/core/scripts/android/build-lz4
```

Comment:
Are we replacing `TARGETS` that was set 2 lines above?

Reply:
Nope. But keep in mind that TARGETS variable is expected inside those scripts. It's not possible to set once and pass to both scripts without actually doing export.